nengo/neurons.py

```diff
@@ -298,8 +300,9 @@ class LIF(LIFRate):
 
     min_voltage = NumberParam('min_voltage', high=0)
 
-    def __init__(self, tau_rc=0.02, tau_ref=0.002, min_voltage=0):
-        super(LIF, self).__init__(tau_rc=tau_rc, tau_ref=tau_ref)
+    def __init__(self, tau_rc=0.02, tau_ref=0.002, amplitude=1, min_voltage=0):
```

Comment:
Technically this breaks backwards compatibility if anyone provided `min_voltage` as a positional argument. But I'm fine with that, I think `min_voltage` was rarely used if at all.